README.rst

```diff
@@ -30,10 +30,14 @@ Usage
     tutor plugins enable discovery mfe credentials
     tutor local launch
 
-Using Django Admin
-~~~~~~~~~~~~~~~~~~
+Creating A Superuser
+~~~~~~~~~~~~~~~~~~~~
 
-The credentials user interface will be available at http://credentials.local.overhang.io for a local instance, and at ``CREDENTIALS_HOST`` (by  default: ``http(s)://credentials.<your lms host>``) in production. In order to run commands from the UI login with an admin user at: http://credentials.local.overhang.io/admin/. User should be able to authenticate with the same username and password that he used for his lms.
```

Comment:
It was working for me and Also for the others in different project.
But for keeping things simple, this command is added in the readme in this [PR](https://github.com/overhangio/tutor-credentials/pull/11) that is merged.

@Talha-Rizwan please close this PR.